flake.nix

```diff
@@ -404,7 +411,8 @@
         dockerImage = buildDocker packages.nickel; # TODO: docker image should be a passthru
         inherit vscodeExtension;
         inherit userManual;
-        inherit stdlibDoc;
+        stdlibDoc = stdlibDoc' "markdown";
```

Comment:
What about renaming `stdlibDoc'` to just `stdlibDoc`, and this output to `stdlibMarkdown`? It just feels a bit more uniform. But just a suggestion.

Reply:
That was just for keeping the packages that the flake exposes consistent. If that's not important here, I'd like `stdlibMarkdown` better, too.

Reply:
I personally think we can do with such a breakage for now (I don't expect anyone to actually use this output outside of us, but I really don't know).